### PR TITLE
Feature/idva6 1074 new invitations endpoints

### PR DESCRIPTION
**Pull request by bmchugh-ch**
Link to Jira:

https://companieshouse.atlassian.net/browse/IDVA6-1074

Changes:

- Integration of get, post /associations/invitations end points

### CODE REVIEW

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

File: src/routers/handlers/yourCompanies/manageAuthorisedPeopleHandler.ts [hunk 1]
```diff
@@ -28,7 +27,7 @@ export class ManageAuthorisedPeopleHandler extends GenericHandler {
         const lang = getTranslationsForView(req.t, constants.MANAGE_AUTHORISED_PEOPLE_PAGE);
         this.viewData = this.getViewData(companyNumber, lang);
         const cancellation: Cancellation = getExtraData(req.session, constants.CANCEL_PERSON);
-        let companyAssociations: Associations = await getCompanyAssociations(req, companyNumber, undefined, undefined, pageNumber - 1);
+        let companyAssociations: AssociationList = await getCompanyAssociations(req, companyNumber);
```
Comment (mmodzelewski-ch) [inline src/routers/handlers/yourCompanies/manageAuthorisedPeopleHandler.ts:30]: Is pageNumber no longer required for getting company associations? How pagination is going to work without this?

Comment (bmchugh-ch) [reply src/routers/handlers/yourCompanies/manageAuthorisedPeopleHandler.ts:30]: Yeah, this looks incorrect. I will update.

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Review by mmodzelewski-ch — APPROVED** [review]

**Pull request merged**